lib/round.rb

```diff
@@ -0,0 +1,70 @@
+class Round
+
+  attr_reader :deck, :card, :turn, :turns, :turn_index, :guesses_correct
+#Initalizes a round object with a argument deck. This has a empty turns
+#array,and a int turns_index.
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @turn_index = 0
+    @guesses_correct = 0
+
+  end
+
+# This function takes a argument guess, and sets it to a turn object.
+#adds that turn object to our turn array and incrementally increases our turn.
+#index by 1. This block also checks if the turn was correct, and adds 1 to the guesses_correct.
+  def take_turn(guess)
+#turn needs a @ because it's just being declared, ask instructor why turns_index
+#breaks without "@" if it's already declared. @turn_index needs "@" because it's
+#being reassigned? (Ask instructor)
+    @turn = Turn.new(guess, current_card)
```

Comment:
You do not need to reference this variable outside of this method, so make it a local variable (without an @ sign)